core/persistence-jpa-json/src/main/java/org/apache/syncope/core/persistence/jpa/dao/PGJPAJSONAnySearchDAO.java

```diff
@@ -18,76 +18,57 @@
-import org.apache.syncope.core.provisioning.api.serialization.POJOHelper;
-import org.apache.syncope.core.persistence.api.entity.JSONPlainAttr;
+import org.apache.syncope.core.persistence.api.entity.Realm;
 import org.apache.syncope.core.provisioning.api.utils.FormatUtils;
+import org.apache.syncope.core.provisioning.api.utils.RealmUtils;
 
 public class PGJPAJSONAnySearchDAO extends AbstractJPAJSONAnySearchDAO {
 
-    @Override
-    protected void processOBS(
-            final SearchSupport svs,
-            final Set<String> involvedPlainAttrs,
-            final OrderBySupport obs,
-            final StringBuilder where) {
-
-        Set<String> attrs = obs.items.stream().
-                map(item -> item.orderBy.substring(0, item.orderBy.indexOf(" "))).collect(Collectors.toSet());
-
-        obs.views.forEach(searchView -> {
-            if (searchView.name.equals(svs.field().name)) {
-                StringBuilder attrWhere = new StringBuilder();
-                StringBuilder nullAttrWhere = new StringBuilder();
-
-                where.append(", (SELECT * FROM ").append(searchView.name);
-
-                if (svs.nonMandatorySchemas || obs.nonMandatorySchemas) {
-                    attrs.forEach(field -> {
-                        if (attrWhere.length() == 0) {
-                            attrWhere.append(" WHERE ");
-                        } else {
-                            attrWhere.append(" OR ");
-                        }
-                        attrWhere.append("plainAttrs @> '[{\"schema\":\"").append(field).append("\"}]'::jsonb");
-
-                        nullAttrWhere.append(" UNION SELECT DISTINCT any_id,").append(svs.table().alias).append(".*, ").
-                                append("'{\"schema\": \"").
-                                append(field).
-                                append("\"}'::jsonb as attrs, '{}'::jsonb as attrValues").
-                                append(" FROM ").append(svs.table().name).append(" ").append(svs.table().alias).
-                                append(", ").append(svs.field().name).
-                                append(" WHERE ").
-                                append("any_id NOT IN ").
-                                append("(SELECT distinct any_id FROM ").
-                                append(svs.field().name).
-                                append(" WHERE ").append(svs.table().alias).append(".id=any_id AND ").
-                                append("plainAttrs @> '[{\"schema\":\"").append(field).append("\"}]'::jsonb)");
-                    });
-                    where.append(attrWhere).append(nullAttrWhere);
-                }
-
-                where.append(')');
-            } else {
-                where.append(',').append(searchView.name);
-            }
-            where.append(' ').append(searchView.alias);
-        });
-    }
+    protected static final String EMPTY_QUERY = "1=2";
```

Comment:
There is already a constant defined in superclass (`JPAAnySearchDAO`) with same name, please rename.